perllib/FixMyStreet/Cobrand/CentralBedfordshire.pm

```diff
@@ -0,0 +1,24 @@
+package FixMyStreet::Cobrand::CentralBedfordshire;
+use parent 'FixMyStreet::Cobrand::Whitelabel';
+
+use strict;
+use warnings;
+
+sub council_area_id { return 21070; }
```

Comment:
Perl uses the value of the last expression in a function as the return value in the absence of the `return` keyword, so this (and the ones below) could more concisely be
```suggestion
sub council_area_id { 21070 }
```

Reply:
I just copied another cobrand file ;-) Looks like about 50% of them use returns, and 50% don’t.